python/lsst/ts/externalscripts/maintel/take_rotated_comcam.py

```diff
@@ -0,0 +1,186 @@
+        super().__init__(
+            index=index, descr="Take AOS sequence at rotated positions with ComCam."
+        )
+        self.angles = None
+        # Timeout for slewing (in seconds).
+        self.slew_timeout = 240.0
+
+    @classmethod
+    def get_schema(cls):
+        """Get the configuration schema, combining base class schema with
+        additional properties."""
+        schema_dict = super().get_schema()
+        schema_dict["$id"] = (
+            "https://github.com/lsst-ts/ts_externalscripts/maintel/TakeRotatedComCam.yaml"
+        )
+        schema_dict["title"] = "TakeRotatedComCam v1"
+        schema_dict["description"] = "Configuration for TakeRotatedComCam."
+
+        additional_schema_yaml = """
+        properties:
+          ra:
+            description: ICRS right ascension (hour).
+            anyOf:
+              - type: number
+                minimum: 0
+                maximum: 24
+              - type: string
+          dec:
+            description: ICRS declination (deg).
+            anyOf:
+              - type: number
+                minimum: -90
+                maximum: 90
+              - type: string
+          target_name:
+            description: Name of the target object.
+            type: string
+          angles:
+            description: Sequence of rotation angles to move the rotator to.
+            type: array
+            items:
+              type: number
+          slew_timeout:
+            description: Timeout for slew operations (seconds).
+            type: number
+            default: 240.0
+        required:
+          - ra
+          - dec
+          - target_name
+          - angles
+        """
+        additional_schema = yaml.safe_load(additional_schema_yaml)
+
+        schema_dict["properties"].update(additional_schema["properties"])
+
+        schema_dict.setdefault("required", [])
+        schema_dict["required"].extend(additional_schema.get("required", []))
+
+        return schema_dict
+
+    def set_metadata(self, metadata):
+        super().set_metadata(metadata)
+        metadata.coordinateSystem = MetadataCoordSys.ICRS
+        radec_icrs = ICRS(
+            Angle(self.config.ra, unit=astropy.units.hourangle),
+            Angle(self.config.dec, unit=astropy.units.deg),
+        )
+        metadata.position = [radec_icrs.ra.deg, radec_icrs.dec.deg]
+
+        # Since we are defaulting to RotType.Physical, metadata rotation system
+        # should be MOUNT
+        metadata.rotationSystem = MetadataRotSys.MOUNT
+        metadata.cameraAngle = float(self.angles[0])
+        metadata.summary = f"Rotator angles: {self.angles}"
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+        await super().configure(config)
+        self.config = config
+        self.angles = config.angles
```

Comment:
you are already storing the entire `config` object, I don't think you need to also store `angles` as a separate attribute.